src/relayserver/TxStoreManager.ts

```diff
@@ -66,7 +65,7 @@ export const TXSTORE_FILENAME = 'txstore.db'
 export class TxStoreManager {
   private readonly txstore: AsyncNedb<any>
   constructor ({ workdir = '/tmp/test/', inMemory = false }) {
-    this.txstore = new Nedb({
+    this.txstore = new AsyncNedb({
```

Comment:
not realy equivalent. one is default, the other is named member

Reply:
Not sure about that 

import AsyncNedb from 'nedb-async'
and 
const Nedb = require('nedb-async').AsyncNedb

seem to use the same class to me. If not, this is extremely confusing.

Also, tests pass.